TestsPython/tests/game_play_test.py

```diff
@@ -37,7 +44,8 @@ def test_game_can_be_paused_and_stopped(self):
     def test_avoiding_obstacles(self):
         self.game_play_page.avoid_obstacles(5)
     # check that player has 2 or 3 lives left (sometimes 1 life is lost right when stopping the tests)
-        assert_that(self.game_play_page.get_current_life()).described_as("player current life").is_greater_than(0) 
+        assert_that(self.game_play_page.get_current_life()).described_as(
```

Comment:
If we want to use pytests we can remove `assert_that` and use the `assert` keyword.

Reply:
I think the `assert` keyword is specific to Python itself. `assert_that` have a lot of built-in functions and it can be useful in the testing process. Should we still give up on `assert_that`? Can it be risky if we use it?

Reply:
yes, `assertpy` is the library and it's just a python library for fluent assertions. We can keep it if we want, it might be a good example for some people.

Reply:
`pytest` overwrite the `assert` statement (https://docs.pytest.org/en/7.4.x/how-to/assert.html#assert) and it makes the error messages more detailed, but if we want we can keep `assertpy`.